src/models/ServerStyleSheet.js

```diff
@@ -48,6 +48,18 @@ class ServerTag implements Tag {
     return `<style type="text/css" ${namesAttr} ${localAttr}>\n${css}\n</style>`
   }
 
+  toElement() {
+    const attributes = {
+      [SC_ATTR]: this.names.join(' '),
+      [LOCAL_ATTR]: this.isLocal,
```

Comment:
We're searching for `el.getAttribute(LOCAL_ATTR) === 'true'`. I'm not sure whether this works, if React just adds the property shorthand; Can you check please? :)